ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -2370,11 +2376,18 @@ snmp_server:
   users:
     - name: < username >
       group: < group_name >
+      # remote_ip and udp_port are used for remote users
```

Comment:
Maybe we could match the model similar to the engine_ids:
```yaml
...
      remote:
        ip: < hostname_or_ip_of_remote_engine >
        port: < udp_port >
```

Reply:
Addressed in f219642